config/calico_versions.yml

```diff
@@ -9,3 +9,6 @@ components:
     version: master
   flexvol:
     version: master
+  rdtigera/init-container:
```

Comment:
what's the `rd` stand for?

Reply:
My initials. That is my personal docker.io. I plan on changing these next week when we have a hooked up the repo to our build processes.